### DISCUSSION TITLE
Does immrich support external uploads

### DISCUSSION THREAD

**Discussion by Riprock** [immich-app/immich#12846] [Q&A] [answered]
Say for instance I host an event and I want attendees to have a link in which they can go to, to see the photos but also upload their own to the album. Is this a current or planned feature?

**Comment by danieldietzler** [comment, marked as answer]
You can create a shared link to an album that allows public users to upload assets, yes.
<img width="409" alt="Screenshot 2024-09-22 at 16 49 35" src="https://github.com/user-attachments/assets/17065e64-33fe-48d8-84de-3b53aa7edfda">

**Reply by Riprock** [reply]
Awesome Thank you! I must have missed that reading the docs

**Reply by danieldietzler** [reply]
It's well possible that's not even in the docs 😅. If you are interested you can play around with our demo instance though: https://demo.immich.app :)
